dask/array/core.py

```diff
@@ -2757,14 +2800,28 @@ def view(self, dtype=None, order="C"):
             chunk.view, dtype, order=order, dtype=dtype, chunks=chunks
         )
 
-    @derived_from(np.ndarray)
     def swapaxes(self, axis1, axis2):
+        """Return a view of the array with `axis1` and `axis2` interchanged.
+
+        Refer to :func:`dask.array.swapaxes` for full documentation.
+
+        See Also
+        --------
+        dask.array.swapaxes : equivalent function
+        """
         from dask.array.routines import swapaxes
 
         return swapaxes(self, axis1, axis2)
 
-    @derived_from(np.ndarray)
     def round(self, decimals=0):
+        """Return `a` with each element rounded to the given number of decimals.
```

Comment:
```suggestion
        """Return `a` with each element rounded to the given number of decimals.
```

Reply:
Actually, we should use `self` instead of `a`

Reply:
Nice suggestion -- I think `array` is more clear 👍

Reply:
```suggestion
        """Return array with each element rounded to the given number of decimals.
```